synapse/config/consent_config.py

```diff
@@ -42,6 +42,14 @@
 # until the user consents to the privacy policy. The value of the setting is
 # used as the text of the error.
 #
+# 'require_at_registration', if enabled, will add a step to the registration
+# process, similar to how captcha works. Users will be required to accept the
+# policy before their account is created.
+#
+# 'policy_name' is the name of the policy users will see when registering for
+# an account. Defaults to "Privacy Policy" and requires require_at_registration
```

Comment:
when you say "requires require_at_registration to be enabled", you mean "will be ignored unless require_at_registration is enabled" rather than "will explode in a heap unless require_at_registration is enabled", right? It would be good to be clearer :)